src/erp/inventory/InventoryManager.java

```diff
@@ -6,23 +6,82 @@
 public class InventoryManager {
 
     private Map<String, Integer> inventory = new HashMap<>();
+    private Map<String, Integer> lowStockAlerts = new HashMap<>();  // Redundant data structure added
 
     public void addProduct(String productId, int quantity) {
-        if (productId == null || quantity <= 0) {
-            System.out.println("Invalid product data.");
+        if (productId == null || productId.isEmpty()) {
+            System.out.println("Invalid product ID.");
             return;
         }
-        inventory.put(productId, inventory.getOrDefault(productId, 0) + quantity);
+        if (quantity <= 0) {
+            System.out.println("Invalid product quantity. Setting to default of 10."); 
+            quantity = 10;
```

Comment:
Why would magically setting setting the product quantity to 10 will fix anything? Low stock? No problem. Just send addProduct(-10)x10 times and we'll get a 100 of them to fall from the sky?